packages/testing/src/chai-retry-plugin/chai-retry-plugin.ts

```diff
@@ -68,29 +68,33 @@ export const chaiRetryPlugin = function (_: typeof Chai, utils: Chai.ChaiUtils)
         const assertionProxy: PromiseLikeAssertion = Object.assign(
             new Proxy(proxyTarget, {
                 get: function (target: Chai.Assertion, key: string, proxySelf: Chai.Assertion) {
-                    if (assertionPropertyKeys.includes(key)) {
-                        assertionStack.push({ property: key });
-
-                        return proxySelf;
+                    let value: Chai.Assertion | undefined = undefined;
+                    try {
+                        value = target[key as keyof Chai.Assertion] as Chai.Assertion;
+                        // eslint-disable-next-line
```

Comment:
how this may throw error? I expect that if there is no such prop, value would be undefined?

Reply:
@olehrakwix  Some properties are actually getters that may perform assertion immediately, thats why we need this `try {} () catch`